# HG changeset patch
# User Rik <rik@octave.org>
# Date 1475078246 25200
#      Wed Sep 28 08:57:26 2016 -0700
# Node ID 29dab5b8332ac5c99f2b7421ce6f58d1f2f89386
# Parent  6763f91685daf9e354008a0bc8dd6d9c6cf1f329
system.tst: Add tolerance of 2*eps to cputime() tests (bug #49166).

* system.tst: Add tolerance (2 * eps (t_expected)) to cputime tests.

diff --git a/test/system.tst b/test/system.tst
--- a/test/system.tst
+++ b/test/system.tst
@@ -17,22 +17,21 @@
 ## <http://www.gnu.org/licenses/>.
 
 %!test
 %! [t1, u1, s1] = cputime ();
 %! for i = 1:200
 %!   sin (i);
 %! endfor
 %! [t2, u2, s2] = cputime ();
-%! assert (t1, u1 + s1);
-%! assert (t2 == u2 + s2);
+%! assert (t1, u1 + s1, 2*eps (t1));
+%! assert (t2, u2 + s2, 2*eps (t2));
 %! assert (t2 >= t1);
 %! assert (u2 >= u2);
 %! assert (s2 >= s2);
-%!#assert (t1 == u1 + s1 && t2 == u2 + s2 && t2 >= t1 && u2 >= u2 && s2 >= s2);
 
 %!test
 %! tic ();
 %! pause (2);
 %! assert (toc () > 0);
 
 %!test
 %! pause (0);
